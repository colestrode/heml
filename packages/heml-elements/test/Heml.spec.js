const render = require('@heml/render')
const Heml = require('../build/Heml').default

test('renders with some attrs', () => {
  const attrs = {
    margin: '1px 2px 3px 4px'
  }

  return expect(render.renderElement(Heml, attrs, '<head></head><body></body>')).toMatchSnapshot()
});
