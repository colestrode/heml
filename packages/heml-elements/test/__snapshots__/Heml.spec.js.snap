// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with some attrs 1`] = `"<!DOCTYPE html PUBLIC \\"-//W3C//DTD XHTML 1.0 Transitional //EN\\" \\"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\\" /><html margin=\\"1px 2px 3px 4px\\" lang=\\"en\\" xmlns=\\"http://www.w3.org/1999/xhtml\\" xmlns:v=\\"urn:schemas-microsoft-com:vml\\" xmlns:o=\\"urn:schemas-microsoft-com:office:office\\"><head></head><body></body></html>"`;
